babel.js

```diff
@@ -37,33 +19,32 @@ module.exports = function (babel) {
     var condition = _.find(attributes, function (attr) {
       return attr.name.name === 'condition';
     });
-
     if (!condition) {
       throwError(errors.IF_WITH_NO_CONDITION, node, file);
     }
 
-    var children = removeLiterals(node.children);
-
-    var notElseTag = function (child) {
-      return child.type !== 'JSXElement' || child.openingElement.name.name !== 'Else';
-    };
-
-    var ifBlock, elseBlock;
+    // normalize JSXText and JSXExpressionContainer to expressions
+    var children = t.react.buildChildren(node);
     ifBlock = _.takeWhile(children, notElseTag);
 
     if (ifBlock.length > 1) {
       throwError(errors.MULTIPLE_CHILDREN, node, file);
     } else if (ifBlock.length === 0) {
       throwError(errors.NO_CHILDREN, node, file);
+    } else {
+      ifBlock = ifBlock[0];
     }
+    //ifBlock = ifBlock.length > 1 ?  t.ArrayExpression(ifBlock) : ifBlock[0];
```

Comment:
ahh, it was a reminder. It's possible to allow for more than one child within control statements. Consequence: We would need an array which leads to extra `<span>`s in some circumstances and React wants `key`s for every item, which could be automatically generated...